sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -458,17 +458,17 @@ describe("receive and delete", () => {
       const deferredMsg = await testDeferredMessage(testClienttype);
       // we have to force this cast - the type system doesn't allow this if you've chosen receiveAndDelete
       // as your lock mode.
-      const msg = deferredMsg as ServiceBusReceivedMessageWithLock;
+      const msg = deferredMsg as ServiceBusReceivedMessage;
```

Comment:
No 'as' needed here anymore.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11962/commits/11c467b6ee3d2d44d0c781d14e9b172d1a56ac0d